infrastructure/router/router.go

```diff
@@ -23,7 +23,16 @@ func New(c controller.NewJobController) *Router {
 
 // InitRouter function
 func (router *Router) InitRouter() *mux.Router {
-	r := mux.NewRouter().StrictSlash(true)
+	r := mux.NewRouter()
+	r.Methods(http.MethodGet).
+		Path("/concurrency/jobs/{type}").
+		Queries("items_per_worker", "{[0-9]+}").
+		Queries("items", "{[0-9]+}").
+		HandlerFunc(router.controller.GetJobsConcurrently)
+	r.Methods(http.MethodGet).
+		Path("/concurrency/jobs/{type}").
+		Queries("items", "{[0-9]+}").
+		HandlerFunc(router.controller.GetJobsConcurrently)
 	r.HandleFunc("/jobs", router.controller.GetJobs).Methods("GET")
 	r.HandleFunc("/api/jobs", router.controller.GetJobsFromAPI).Methods("GET")
```

Comment:
```suggestion
	r.HandleFunc("/jobs", router.controller.GetJobs).Methods(http.MethodGet)
	r.HandleFunc("/api/jobs", router.controller.GetJobsFromAPI).Methods(http.MethodGet)
```
No magic naming; you used the already defined constant in the two endpoints above.